### PR TITLE
Rename [some] Genesis stuff to no-intro 2016-10-23.

### PR DESCRIPTION
**Pull request by sergiobenrocha2**
Purged the beta & proto box arts too, they are probably fake anyway

### CODE REVIEW

**Comment by lasers** [comment]
Sharing my findings.

`Sega - Mega Drive - Genesis (20161023-001129_CM).dat` is not in the database. It might be a good idea to get that database merged in first so everybody always works with the same database.

These fake box arts might not be fake at all. I don't have `libretro-thumbnails` here so I only checked few files. I can confirm 103 Beta zips for Genesis. Also, both old and the new database contains just `(Beta)` and `(Beta) (Date/Alt 1/Unl/Earlier/$DATE)` too.

Why delete if it's still in the database?

**Comment by sergiobenrocha2** [comment]
I deleted only the beta/proto box arts, I'll not mess with the database, it does not make any sense... btw it was a non-sense to remove bios entries there, ihmo.

Those beta/proto games were not officially released in the market, so it does not make any sense to maintain box arts for them. There are even fake box arts for official games (see https://github.com/libretro/libretro-thumbnails/issues/6#issuecomment-214046569), which we are going to change over time.

**Comment by lasers** [comment]
Removing `[BIOS]*` only makes it hard for people if for any reason they might want to look at them, they pretty much had to do it manually just because the list is filtered. I did it 10~ times just now.

And because the list is filtered right away from the no-intro dat, I have to ask what's stopping the team from filtering `(Beta)` or `(Unl)`? My opinion is that the team should not modify the database so much and to keep everything genuine as possible (straight from the source). If I have (and I do!) the `[BIOS]*`, I should be able to view them in the list and be able to open them from the list too... even if I'm not planning on doing that again... Just like some games I tried. 

Instead of "hardcoding" database to remove certain files, we should consider seeing if it's in our wishes to add flexible/customizable filters. One could filter `USA` to get all USA and probably some Europe/Japan "USA" files... or to add filter `[BIOS]` to hide all `[BIOS]*` files. If you don't like that or think it's too much, then we shouldn't scrub anything off the list in the first place.

I looked at all of `[BIOS]*` for Genesis. While they may not have much purpose, they're still nice to look at... for nostalgia... or to mess with somebody by displaying Dreamcast startscreen on Wii or Vita...  after all, I did download them. If people don't want to see `[BIOS]*` in their list, they should remove it from their `Genesis` folder just like one might do with all foreign games. https://sli.mg/a/zI0NFs

I understand that this might be a team thing. Just something for the team to consider. RetroArch is programmed to work with many platforms as possible yet there are "hardcoded" filters to weed out some files. More than a team probably thought it was ugly, useless, or unwanted. I can agree with that too, but they likely would work without any issues... even if the files does not have much purpose right now... So does the bad games too. 

This goes same for beta or unlicensed, games, artworks, cheats, etc. If it's in the no-intro database, we should leave the files alone cause I do still haz my betas. https://sli.mg/kqwlII 

tl;dr: It just sucks that I can't view the BIOS in the list because the team says so... and not to remove thumbnails because it is not an official game.

Have a happy Thanksgiving! :-)

**Comment by sergiobenrocha2** [comment]
I removed beta/proto box art because they DIDN'T EXIST at all in the real world. Snaps and Titles are there yet.
